configsnap

```diff
@@ -72,8 +76,23 @@ def create_dir(tagdir, overwrite):
     return workdir
 
 
-diffs_found_msg = False
-is_php_detected = False
+def create_archive(tagdir, workdir, tag, overwrite):
+    """Create tar archive of workdir and return filename and success"""
+    output_filename = "%s/configsnap-%s.tar.gz" % (tagdir, tag)
+
+    if os.path.isfile(output_filename):
+        if overwrite:
+            report_info(
+                "%s exists. Overwrite (-w/--overwrite) enabled so will overwrite." % output_filename)
+        else:
+            report_error(
+                "Archive file %s already exists. Use -w overwrite." % output_filename)
+            return "", False
+
+    with tarfile.open(output_filename, "w:gz") as tar:
```

Comment:
Having the syntax in the file itself is causing it to error out. Pinging @thomas-rudd for more details.